File: JAVA/Bit_Manipulation/Clear_Bit.java
import java.util.*;

// Clear the 3rd bit (position = 2 ) of a number n.
//Answer : - 1. BitMask : 1<<postion 2.AND with NOT.
public class Clear_Bit {
    public static void main(String[] args) {
        int n = 5;
        int pos = 2;
        int bitmask = 1 << pos;
        int notbitmask = ~(bitmask);

        int newNumber = n & notbitmask;

        System.out.println(newNumber);
    }
}

File: C++/Bit_manipulation/Clear_Bit.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n = 5 ;
    int position = 2;
    int bitmask = 1<<position;

    int newBitmask = ~(bitmask);
    int newNumber = n & newBitmask;

    cout<<newNumber<<endl;
}